app/models/engineering_search_builder.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+# This is a demonstration search builder, not intended for prodcution use.
+class EngineeringSearchBuilder < SearchBuilder
+  self.default_processor_chain += [:switch_request_handler]
+
+  def switch_request_handler(solr_parameters)
+    solr_parameters[:qt] = "engineering_search"
```

Comment:
I like this `qt` approach!  I wonder if still works in the following specific case:

1. The JSON Query DSL feature is enabled
2. The View Components Advanced Search feature is enabled
3. The user does a search in the advanced search screen, and switches to the "engineering" requesthandler

If I remember correctly, those circumstances send queries to the `advanced` requesthandler, rather than the default one, where I don't know that `qt` would have an effect.